docs/testing.md

```diff
@@ -0,0 +1,138 @@
+# Testing BMI
+
+Every Developer will need to test his/her feature or make sure that he/she
+did not break anything. So the test cases are situated in
```

Comment:
"Every developer will need to test **their** feature or make sure that **they** did not break anything" 
Just minor nitpick :) because it sounds better this way.